src/indexing/index.py

```diff
@@ -81,6 +84,10 @@ def decache_token(self, token: str):
         if ltoken in self._query_cache:
             del self._query_cache[ltoken]
 
+    def _load_citation_data(self) -> None:
+        with open(str(self._pubmed_dir) + "/test.json", encoding="utf-8") as f:
```

Comment:
check to see if file exists before opening it. and need to have some code that builds this .json file somewhere

Reply:
rename the json file something meaningful, and store the name somewhere so you can reference it elsewhere. so you don't need to put '/test.json' in unit tests, etc.